mat-process-api/V1/Controllers/ProcessDataController.cs

```diff
@@ -54,21 +58,41 @@ public IActionResult GetProcessData(string processRef)
         /// If processData needs to be update, it's instead replaced with a full new one.
         /// Returning flat out 200 on successful update
         /// </summary>
-        /// <param name="processDataJSON"></param>
+        /// <param name="updateRequest"></param>
         /// <returns></returns>
         [HttpPatch]
+        [Route("{processRef}")]
```

Comment:
Is this parameter supposed to be captured or used anywhere? (Asking because it's not used)